sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/tests/EventHubEndToEndTests.cs

```diff
@@ -198,6 +198,42 @@ public async Task CanSendAndReceive_TokenCredentialInConfiguration()
                     })));
         }
 
+        [Test]
+        public async Task CanSendAndReceive_BlobServiceUri_InConfiguration()
+        {
+            await AssertCanSendReceiveMessage(host =>
+                host.ConfigureAppConfiguration(configurationBuilder =>
+                    configurationBuilder.AddInMemoryCollection(new Dictionary<string, string>()
+                    {
+                        {"TestConnection:fullyQualifiedNamespace", EventHubsTestEnvironment.Instance.FullyQualifiedNamespace},
```

Comment:
Shouldn't this be "TestConnection:serviceUri"? Just thought all of track2 was moving to this naming scheme for configuration.

Reply:
We are mapping the setting name to the client constructor parameter name 

https://apiview.dev/Assemblies/Review/e9236fb0f0e34c40bf72df92acddfed3#Azure.Messaging.EventHubs.Primitives.EventProcessor%3CTPartition%3E
 
=>

fullyQualifiedNamespace

 

https://apiview.dev/Assemblies/Review/77426af3f12a4487ab98859fca53a6dc#Azure.Storage.Blobs.BlobServiceClient

=> 

serviceUri